package.json

```diff
@@ -9,15 +9,11 @@
     "ipython",
     "kernel"
   ],
-  "homepage": "https://github.com/Quansight/ksmm",
+  "homepage": "https://github.com/deshaw/ksmm",
   "bugs": {
-    "url": "https://github.com/Quansight/ksmm/issues"
+    "url": "https://github.com/deshaw/ksmm/issues"
   },
   "license": "BSD-3-Clause",
-  "author": {
```

Comment:
This breaks the build. Similar to jupyterlab-pyflyby, let's just remove author from setup.py as well

Reply:
Thx for the review. I have removed the authors from setup.py and now CI is green.